src/Workspaces/Remote/ServiceHub/ExternalAccess/UnitTesting/Api/UnitTestingServiceHubBase.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Remote;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.UnitTesting.Api
+{
+    internal abstract class UnitTestingServiceHubBase : ServiceHubServiceBase
```

Comment:
📝 If a base type is not accessible, then the type itself is not accessible. This _shouldn't_ have compiled successfully; if it did then please file a bug on dotnet/roslyn-analyzers so we can correct it.

Reply:
I don’t think we have any analyzer to validate the changes to Roslyn side.

Reply:
Here is the issue to fix that: https://github.com/dotnet/roslyn-analyzers/issues/2648